### DISCUSSION TITLE
your doc

### DISCUSSION THREAD

**Discussion by ginolegigot** [danymat/neogen#200] [General]
your doc is amazing thanks

**Comment by danymat** [comment]
Oh wow, thanks a lot ! What did you like specially ? 
I didn't update the docs for quite some time, but I guess they are still relevant to the neon code

**Reply by ginolegigot** [reply]
You have what i look for in every nvim plugins, gifs to understand in a glance/visually the plugin purposes and main capabilities. Many examples of different working configurations. It just works.
Thanks.
